.github/workflows/build.yaml

```diff
@@ -42,49 +42,32 @@ jobs:
         script: |
           arm-none-eabi-gcc --version
           qemu-system-arm --version
-    - name: Emulate on QEMU
+    - name: Build for emulation on QEMU
       shell: nix develop .#ci -c bash -e {0}
       run: |
-          tests=10
-          output=$(make emulate NTESTS="$tests")
-          echo "::group::make emulate"
-          echo "$output"
-          echo "::endgroup::"
-
-          for file in elf/*.elf; do
-            echo "::group::Emulate $file"
-
-            output=$(make emulate ELF_FILE="$file")
-            echo "$output"
-
-            # check if error occurred
-            err=$(echo "$output" | awk '/ERROR/{code=1;print $2} END {exit code}')
```

Comment:
originally, since I run the bash with `-e` flag, any error in a subshell will be captured, then the whole shell will exit immediately